<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>399</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>474</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="pic_label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>-6</y>
     <width>231</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="agree_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disagree_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>不同意</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
